src/components/pac/__tests__/PacForm.spec.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
```

Comment:
As we are migrating from ezyme test to React testing library, Its better to migrate the enzyme unit tests to RTL while moving to console-plugin.